AndroidSDKCore/src/main/java/com/leanplum/internal/Request.java

```diff
@@ -230,7 +221,7 @@ public void onApiResponse(ApiResponseCallback apiResponse) {
     return args;
   }
 
-  private void saveRequestForLater(final Map<String, Object> args) {
+  private void saveRequestForLater(final Map<String, Object> args, final boolean sendNow) {
```

Comment:
This method should do 1 thing. If you need to do something afterwards, perhaps you can pass a callback to run additional code when this finishes?